napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -1178,13 +1187,34 @@ def _setup_ui(self):
         visibility_direct_entry = not running_as_constructor_app()
         self.direct_entry_edit = QLineEdit(self)
         self.direct_entry_edit.installEventFilter(self)
-        self.direct_entry_edit.setPlaceholderText(
-            trans._('install by name/url, or drop file...')
-        )
+        self.direct_entry_edit.returnPressed.connect(self._install_packages)
         self.direct_entry_edit.setVisible(visibility_direct_entry)
-        self.direct_entry_btn = QPushButton(trans._("Install"), self)
+        self.direct_entry_btn = QToolButton(self)
         self.direct_entry_btn.setVisible(visibility_direct_entry)
         self.direct_entry_btn.clicked.connect(self._install_packages)
+        self.direct_entry_btn.setText(trans._("Install"))
+        self.direct_entry_btn.setToolButtonStyle(Qt.ToolButtonTextBesideIcon)
```

Comment:
Setting the popupMode of the toolbutton a behavior were you click the arrow beside the icon can be achived. So changing things here to be something like:

```suggestion
        self.direct_entry_btn.setPopupMode(QToolButton.MenuButtonPopup)
```

With that behave something like this (at least on Windows):

![install_toolbutton_arrow_click](https://github.com/user-attachments/assets/784a499c-ba7d-4a5d-8ada-822ea5fdc57d)